pypistats/__init__.py

```diff
@@ -166,8 +166,42 @@ def _custom_list(input_list, special_item, default_value, special_value):
     return new_list
 
 
-def _tabulate(data):
-    """Return data in table"""
+def _tabulate(data, format="markdown"):
+    """Return data in specified format"""
+
+    format_methods = {"markdown": _tabulate_markdown, "rst": _tabulate_rst}
+
+    return format_methods[format](data)
+
+
+def _tabulate_rst(data):
```

Comment:
There's only one line different between `_tabulate_rst` and `_tabulate_markdown`:

![image](https://user-images.githubusercontent.com/1324225/47792481-cfd1e280-dd24-11e8-8c80-14492e67722d.png)

How about retaining `_tabulate` and putting something like this at the top?

```python
    writer_methods = {"markdown": MarkdownTableWriter, "rst": RstSimpleTableWriter }
    writer = writer_methods[format]()
```

Reply:
Never thought until you pointed.
will do